stravalib/model.py

```diff
@@ -997,3 +998,51 @@ class Route(LoadableEntity):
     # timestamp = NOT IMPLEMENTED
     # segments = NOT IMPLEMENTED
 
+
+class Subscription(LoadableEntity):
+    """
+    Represents a Webhook Event Subscription.
+
+    http://strava.github.io/api/partner/v3/events/
+    """
+    OBJECT_TYPE_ACTIVITY = 'activity'
+    ASPECT_TYPE_CREATE = 'create'
+
+    VERIFY_TOKEN_DEFAULT = 'STRAVA'
+
+    application_id = Attribute(unicode, (SUMMARY, DETAILED))
+    object_type = Attribute(unicode, (SUMMARY, DETAILED))
+    aspect_type = Attribute(unicode, (SUMMARY, DETAILED))
+    callback_url = Attribute(unicode, (SUMMARY, DETAILED))
+    created_at = TimestampAttribute((SUMMARY, DETAILED))
+    updated_at = TimestampAttribute((SUMMARY, DETAILED))
+
+
+class SubscriptionCallback(LoadableEntity):
+    """
+    Represents a Webhook Event Subscription Callback.
+    """
+    def __init__(self, *args, **kwargs):
+        """
+        Special/complicated __init__ because of tricky variable names
+        """
+        setattr(self, 'hub.mode', Attribute(unicode, (SUMMARY, DETAILED)))
+        setattr(self, 'hub.verify_token', Attribute(unicode, (SUMMARY, DETAILED)))
+        setattr(self, 'hub.challenge', Attribute(unicode, (SUMMARY, DETAILED)))
```

Comment:
So, I think I see what's going on here ,but I'm wondering if maybe the deserialize() method should be modified to handle "." characters so that we can keep this model definition looking like more traditional models.  I feel like not being able to reference these attributes on the class could be problematic in other areas.

So, for example, change definition to look like:
```python

class SubscriptionCallback(LoadableEntity):
    hub_mode = Attribute(unicode, (SUMMARY, DETAILED))
    # etc.
```

And then change `deserialize()` so that it if detects characters that would be invalid in a python attribute name in the json dictionary keys, it just replaces them with '_'.

Granted, I haven't looked at `deserialize()` code in awhile, but what do you think about that?